Assets/Console.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
```

Comment:
Is there a reason for this file, it seems like you may have made it but didn't put anything in it

Reply:
If it is to attach a component to be found, make sure to remove the functions inside of Start and Update